verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -0,0 +1,262 @@
+openapi: 3.0.3
+info:
+  title: EPID curator service API
+  description: Server for CRUD operations on EPID data
+  termsOfService: "https://www.healthmap.org/about/termsofuse/"
+  contact:
+    email: info@healthmap.org
+  license:
+    name: MIT
+    url: "https://opensource.org/licenses/MIT"
+  version: 1.0.0
+paths:
+  /sources:
```

Comment:
Since we have multiple resources, it'd be nice to add tags for the resource (`Source`; `User`). Then, in the UI, we'd have grouped method definitions by resource. WDYT?

Reply:
+1

Reply:
beautiful!